### PR TITLE
AceVice One's keymap for Kyria keyboard

### PR DESCRIPTION
**Pull request by LuisMorenoSerrano**
## Description

Keymap definition for Kyria Keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kbdfans/kbd75rgb/info.json [hunk 1]
```diff
@@ -7,5 +7,100 @@
...
+                {"label": "Alt", "x":2.5, "y":5, "w":1.25},
+                {"label": "Space", "x":3.75, "y":5, "w":6.25},
+                {"label": "Alt", "x":10, "y":5},
+                {"label": "GUI", "x":11, "y":5},
+                {"label": "Ctrl", "x":12, "y":5},
+                {"label": "Left", "x":13, "y":5},
+                {"label": "Down", "x":14, "y":5},
+                {"label": "Right", "x":15, "y":5}
+            ]
+        }
```
Comment (zvecr) [inline keyboards/kbdfans/kbd75rgb/info.json:104]:
```suggestion
```

Comment (LuisMorenoSerrano) [reply keyboards/kbdfans/kbd75rgb/info.json]: Sorry, commit 3444dd5e2d (PR #18391) was not part of my PR. I have removed it from my branch and PR.

Thanks!

File: keyboards/splitkb/kyria/keymaps/lms_ace01/README.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# AceVice One's Keymap for [Kyria](https://github.com/splitkb/kyria)
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/lms_ace01/README.md:1]: Please rename this file so that it's all lowercase.

Comment (LuisMorenoSerrano) [reply keyboards/splitkb/kyria/keymaps/lms_ace01/README.md]: Done! Thanks for your advice

File: keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c [hunk 3]
```diff
@@ -0,0 +1,852 @@
...
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+    // Get current modifier status (Normal and One Shot)
+    uint8_t mod_state = get_mods();
+    uint8_t osm_state = get_oneshot_mods();
+
+    // Process keycode
+    switch (keycode) {
+        case CK_QUOT:  // Single/Double Quote
+            if (record->event.pressed) {
+                if ((mod_state | osm_state) & MOD_MASK_SHIFT) {
```
Comment (drashna) [inline keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c:409]: Key Overrides can probably handle this better.

**Review by drashna — APPROVED** [review]

File: keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c (hunk 3 above)
Comment (LuisMorenoSerrano) [reply keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c:409]: With spanish keyboard, ES_QUOT sends single quote (') on normal mode and question mark (?) on shifted mode. In addition to this, I would like to have another key with default behaviour: single quote (') on normal mode and double quote (") on shifted mode. This should be as simple as use KC_QUOT keycode, but when spanish keyboard activated it does not work in that way. If I override ES_QUOT with ES_2 when shifted, I would lose spanish behaviour. I didn't find another way to accomplish this.

Comment (drashna) [reply keyboards/splitkb/kyria/keymaps/lms_ace01/keymap.c:409]: Ah, okay, no worries then.

**Review by keyboard-magpie — APPROVED** [review]

**Pull request merged**